### PR TITLE
Contact form addition to portfolio Website

### PR DESCRIPTION
**Pull request by Newtayo**
The following features has been updated on the portfolio mobile website

- Added a contact form section for users to send in their requests.
- The contact form has been integrated with formspree to enable it store information.
- The submit button in the contact form section now has a hover effect

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -510,4 +513,78 @@
...
+    margin: auto;
+    height: 8vh;
+    color: #fff;
+    border: none;
+    background: #ff6b00;
+  }
+
+  .btn:hover {
+    border: 2px solid red;
+    cursor: pointer;
```
Comment (nahnahdev) [inline style.css]: - Great job so far :muscle: Your project is almost complete. There is just something small you have to do to proceed to the next milestone :smile: 
- [ ] I noticed that your button is missing the right button interaction styles that the template you picked provided you with. I have attached a screenshot below :wink: 
- The Figma Design
image
- Your Design in all of these states
image
- [ ] Please try to stick to the Figma design as much as possible. This includes colors, margins, padding etc. :wink:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @Newtayo and @Adholah96, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
- Project well documented :heavy_check_mark: 
- Nice design :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N / A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Newtayo** [comment]
Button Feature such as hover, enabled and disabled button have been added to the buttons

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Formspree working as expected :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**